apps/web/controllers/shared/pagination.rb

```diff
@@ -0,0 +1,15 @@
+module Web
+  module Controllers
+    module Shared
+      module Pagination
+        include Pagy::Backend
+
+        private
+
+        def pagy_get_items(repository, pagy)
```

Comment:
can we call those methods without `pagy` prefix?
It would be nice to hide this implementation detail, and have an abstract interface.